PIPELINE_TEMPLATES.md

```diff
@@ -485,6 +494,56 @@ A couple key things to note here:
    dependencies. Note that the stage `finalWait` stage only depends on 
    `buildChrome.publishTarget`, rather than every stage defined by the Partial.
 
+# Imports
+
+Importing is supported at both the Template and Configuration level and can be
+used to break up a large files into more digestable units, or to share common
+reusable components across multiple Templates. Unlike a Template hierarchy,
+imported Templates _cannot_ define Stages.
+
+Imported Templates share the same file structure as a Template, but some fields
+become ignored:
+
+* `source`: Imports cannot inherit from other templates or other Imports.
+* `imports`: Imports cannot include other Imports.
+* `configuration`: Imports cannot define template configuration settings.
+* `variables`: To prevent leaking too much into the global scope of a Template,
+  Imports cannot define root-scoped Variables.
+* `stages`: Imports are not allowed to define stages.
+
+Imports can be included into a Template or Configuration using the Template
+Loader syntax. The Partials and Modules contained in an Import are automatically
+namespaced onced imported as `{{importId}}.{{partialId|moduleId}}`
```

Comment:
I wonder if you should be able to define some sort of aliasing of your imported modules/partials. I'm envisioning a case where you want to use a parent template, but you want to override a module/partial that it uses with an imported module/partial.

Reply:
Good thought!